### PR TITLE
Upgrade React and Antd till the latest version

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context

- [x] Annotation menu
- [x] OpenCV is not loaded automatically
- [x] Eslint
- [x] Stylelint
- [x] Tests

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-ui/src/components/actions-menu/styles.scss [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-    > li:hover {
-        background-color: $hover-menu-color;
-    }
-
-    .ant-menu-submenu-title {
-        margin: 0;
-        width: 13em;
-    }
-}
+@import '../../base';
```
Comment (klakhov) [inline cvat-ui/src/components/actions-menu/styles.scss:6]:
```suggestion
@import 'base';
```

File: cvat-ui/src/components/actions-menu/styles.scss [hunk 2]
```diff
@@ -1,22 +1,9 @@
...
 
 .cvat-menu-dump-submenu-item,
```
Comment (klakhov) [inline cvat-ui/src/components/actions-menu/styles.scss:8]: I cant find usage of this classes in our code now

File: cvat-ui/src/components/analytics-page/styles.scss [hunk 3]
```diff
@@ -129,7 +130,7 @@
     padding: 6px 8px;
     border: 1px solid lightgray;
     margin-left: 8px;
```
Comment (klakhov) [inline cvat-ui/src/components/analytics-page/styles.scss:132]:
```suggestion
    margin-left: $grid-unit-size;
```

**Review by klakhov — COMMENTED** [review]
Sidepanel became white:
image

**Review by klakhov — COMMENTED** [review]
There is something wrong with initial position of opencv tool popup 
opencvplacement

**Comment by bsekachev** [comment]
@klakhov  Applied

**Review by klakhov — COMMENTED** [review]
Exclamation mark and excessive left padding now appear on organization switch modal. Also I cant close it without selecting something new
image

Do we really want to use blue hover effect? I think it doesn't stand out enough on gray background. Maybe we can use scaling effect instead.
image

Here I suppose using bold black font would be better instead of blue too. 
image

If we dont have an assignee, maybe we dont want to show the line with it at all? Or at least say something like not assigned so we dont have emty space.
I like the idea of showing stage+state in one status row like `annotation in progress` or `annotation rejected`, it looks cool. But it looks strange with `new`, maybe we should use an alias for that something like `annotation not started`. 
image

Small padding problem on models page(it seems the same problem is on the projects page):
image

Major version update for package.json?

**Comment by bsekachev** [comment]
> Exclamation mark and excessive left padding now appear on organization switch modal

I believe this is general antd change: 
image

> Major version update for package.json?

I do not see reasons to update major version. Generally all the changes are cosmetic.

**Comment by bsekachev** [comment]
>  Also I cant close it without selecting something new

Applied

> Do we really want to use blue hover effect?

Default antd button, I reversed it as it was with custom css.

> Here I suppose using bold black font would be better instead of blue too.

The same.

 > If we dont have an assignee, maybe we dont want to show the line with it at all? Or at least say something like not assigned so we dont have emty space.

The main idea is to keep height the same. Sliding bottom part is annoying a little bit. 

> I like the idea of showing stage+state in one status row like annotation in progress or annotation rejected, it looks cool. But it looks strange with new, maybe we should use an alias for that something like annotation not started.

The main idea behind it was to keep vertical space, using horizontal free space. I would avoid customizing logic in described way.

> Small padding problem on models page(it seems the same problem is on the projects page):

Fixed on projects/jobs/models. Also fixed top bars on different resolutions.

**Review by klakhov — APPROVED** [review]
Looks good!

**Pull request merged**